deploy/template.yaml

```diff
@@ -105,6 +137,15 @@ Resources:
         - !GetAtt LoadBalancerSG.GroupId
       Subnets: !Ref SubnetIds
       Type: application
+      LoadBalancerAttributes:
+        - Key: access_logs.s3.enabled
+          Value: true
+        - Key: access_logs.s3.bucket
+          Value: !Ref AccessLogsBucket
+        - Key: access_logs.s3.prefix
+          Value: core-front
```

Comment:
worth adding ${Environment} to the prefix?
I know it's already in an ${Environment} bucket, but might be worth including it